app/components/LaptopMockup.tsx

```diff
@@ -5,12 +5,13 @@ import { laptopTemplate } from "@/public/png-icons/featured-projects-images";
 import { MockupProps } from "@/utils/interfaces";
 
 const laptopMockupStyles = {
-  homepageParent: "h-40 w-64 lg:h-[21rem] lg:w-[35rem]",
-  otherPageParent: "h-[10.6rem] w-[17.9rem] sm:h-[19.5rem] sm:w-[32.8rem]",
+  homepageParent: "min-h-40 min-w-64 lg:h-[21rem] lg:w-[35rem]",
+  otherPageParent:
+    "min-h-[10.6rem] min-w-[17.9rem] sm:h-[19.5rem] sm:w-[32.8rem]",
   homepageChild:
-    "top-[0.18rem] h-[9.22rem] rounded w-[13.26rem] left-[1.432rem] lg:left-12 lg:h-[19.25rem] lg:w-[28.9rem] lg:rounded-t-[10px] lg:top-[0.4rem]",
+    "top-[1.7%] h-[92%] rounded-t-[3%] rounded w-[83%] left-[8.9%] lg:left-[3.1rem] lg:h-[19.25rem] lg:w-[28.78rem] lg:rounded-t-[10px] lg:top-[0.35rem]",
```

Comment:
For a more consistent and maintainable approach, it's generally better to stick to a single unit type for similar properties and use media queries to adjust styling as necessary for different screen sizes. Mixing units in CSS can lead to inconsistent and unpredictable styling across different screen sizes and devices. 

As a side note, in case you weren't already aware, different units like percentages, rems, and pixels are relative to different things. _Percentages_ are relative to the **parent** element, _rems_ are relative to the **root font size**, and _pixels_ are **absolute units**. You can see how this could lead to inconsistent behavior, especially when the styling needs to adapt to different screen sizes.

Reply:
Hi @cdsalty. I have used percentages in this particular instance because, like you wrote, I want this div to be directly relative to the parent element until it reaches large screens in which I am switching everything to a fix width and height. This was the only way so far that I could get this div to be responsive and stay in the exact proportions that I wanted it too while I'm resizing the screen. Before, the image of the laptop mockup stayed one size from the smallest screen until the large screen at which time it switches to a larger fixed size. This was I was able to make it responsive and as such it doesn't look too small on medium screens now.

Reply:
Hi @cdsalty I think I have this fixed better now. I am now longer mixing percentages and rems within the same classname. The child div is now totally managed with percentages and the parent is totally managed with rem. What do you think? Is that better than how I was doing it before?